view-admin/src/pages/index.tsx

```diff
@@ -1,114 +1,96 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import { Inter } from 'next/font/google'
-import styles from '@/styles/Home.module.css'
+import styles from "@/styles/Home.module.css";
+import type { NextPage } from "next";
+import {
+  Header,
+  BingoResult,
+  Button,
+  JudgementModal,
+} from "@/components/common";
+import { CgLogOut } from "react-icons/cg";
+import { useState } from "react";
 
-const inter = Inter({ subsets: ['latin'] })
+const Page: NextPage = () => {
+  const bingoResultNumber: number[] = [
+    21, 5, 33, 1, 1, 1, 1, 1, 1, 1, 2, 2, 2, 2, 2, 2, 3, 3, 3, 3, 3, 4, 4, 55,
+    66, 32,
+  ];
+  const [inputNumbers, setInputNumbers] = useState<number[]>([]);
+  const [isIncluded, setIsIncluded] = useState(false);
+
+  const checkInclusion = () => {
+    const included = inputNumbers.length === 5 && inputNumbers.every(number => bingoResultNumber.includes(number));
+    setIsIncluded(included);
+  };
```

Comment:
```suggestion
  const checkInclusion = () => {
    if (inputNumbers.every(number => bingoResultNumber.includes(number))) {
      setIsIncluded(true);
    }
  };
```
setIsIncludedに直接true入れたら多分いける
あと、lengthが５あるかはmodalのuseStateで５つ定義してるからいらなそう

Reply:
対応しました。